packages/mobile/src/components/add-to-playlist-drawer/AddToPlaylistDrawer.tsx

```diff
@@ -86,17 +96,23 @@ export const AddToPlaylistDrawer = () => {
       title={messages.title}
     >
       <View>
-        <View style={styles.buttonContainer}>
-          <Button
-            title='Create New Playlist'
-            onPress={addToNewPlaylist}
-            containerStyle={styles.button}
-            type={ButtonType.COMMON}
-          />
-        </View>
+        {!isPlaylistUpdatesEnabled ? (
+          <View style={styles.buttonContainer}>
+            <Button
+              title='Create New Playlist'
+              onPress={addToNewPlaylist}
+              containerStyle={styles.button}
+              type={ButtonType.COMMON}
+            />
+          </View>
+        ) : null}
         <CardList
           contentContainerStyle={styles.cardList}
           data={userPlaylists}
+          showCreatePlaylistTile={isPlaylistUpdatesEnabled}
```

Comment:
this works and makes sense, i have thoughts on if we can add the "add playlist" tile as a datum that gets passed to the card list? so instead of card list having to know about playlist-tile, it just gets injected into the playlist card list? def down to pair an see how this might be done :)